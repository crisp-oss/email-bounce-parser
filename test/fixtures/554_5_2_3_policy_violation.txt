This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<support@databit.ch>: host mx.zoho.eu[158.203.221.166] said: 554 5.2.3
    MailPolicy violation Error delivering to mailboxes (in reply to end of DATA
    command)
