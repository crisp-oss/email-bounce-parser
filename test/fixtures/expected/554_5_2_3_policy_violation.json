{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<support@databit.ch>: host mx.zoho.eu[158.203.221.166] said: 554 5.2.3 MailPolicy violation Error delivering to mailboxes (in reply to end of DATA command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<support@databit.ch>: host mx.zoho.eu[158.203.221.166] said: 554 5.2.3 MailPolicy violation Error delivering to mailboxes (in reply to end of DATA command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "554",
        "enhanced": "5.2.3"
      },
      "label": "MailPolicy violation Error delivering to mailboxes",
      "type": "transaction_failed",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": null,
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "support@databit.ch",
    "server": {
      "hostname": "mx.zoho.eu",
      "ip": "158.203.221.166",
      "port": null
    },
    "command": "DATA"
  }
}